### DISCUSSION TITLE
Feedback and potentially actionable directions for the lotus release process

### DISCUSSION THREAD

**Discussion by pooja** [filecoin-project/lotus#5383] [General Q&A]
We've learned a lot over the last few months about how to manage lotus releases and bring our community alongside as we upgrade the network. However, now that we're in a mainnet world, the stakes are higher across the board for our users (miners, partners, clients, devs, etc). To that end, we've been collecting some feedback from the team and from users and stakeholders that could be helpful in drafting a more holistic release process/checklist/playbook for lotus!

The feedback doc is here: https://docs.google.com/document/d/1SaO-KRPvgIMa2wSyDx4n3I55wqtYg6tYpTD0Wq7dfYM/edit#

I'm sure we already have release checklists in progress, but the main areas that are coming up in the feedback so far to include as part of our release process are:
- Release **testing**
- **Comms:** Notifications to the community on official channels
- Detailed **changelogs**, esp describing changes to **APIs**
- **More time** between cutting the release and upgrade epoch

Tagging in @dineshshenoy @arajasek to start the conversation here and hopefully help prep a release playbook that works well for the dev team and for users 🙏

**Comment by dineshshenoy** [comment]
Release testing draft here: https://docs.google.com/document/d/1411_inv17hMZpxhXn_3e5k13AKqdpmPmSh3DGyxgP7E/edit#heading=h.y3v3eeykwihw

**Comment by dineshshenoy** [comment]
Other release ideas:  https://docs.google.com/document/d/19W21Awi7-BCbAziIuGnIeQ50zonVGmKqkWKppXnkgFE/edit#

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
https://github.com/filecoin-project/lotus/issues/4244 we had this a long long time ago. closed it due to inactivity
